playground/app.vue

```diff
@@ -1,27 +1,16 @@
 <template>
   <div class="container">
+    <Meta hid="description" name="description" content="Demo of using @nuxtjs/color-mode for Nuxt, supporting dark mode and custom colors" />
```

Comment:
```suggestion
    <Meta key="description" name="description" content="Demo of using @nuxtjs/color-mode for Nuxt, supporting dark mode and custom colors" />
```

Reply:
Good point! And it's automatically keyed on `name` so we should be fine to drop entirely.
```suggestion
    <Meta name="description" content="Demo of using @nuxtjs/color-mode for Nuxt, supporting dark mode and custom colors" />
```